reana-ui/src/pages/workflowList/components/WorkflowSorting.js

```diff
@@ -12,16 +12,15 @@ import PropTypes from "prop-types";
 import { Dropdown } from "semantic-ui-react";
 
 const sortOptions = [
-  { key: 1, text: "Newest", value: "desc" },
-  { key: 2, text: "Oldest", value: "asc" },
-  { key: 3, text: "Most disk quota used", value: "disk-desc" },
-  { key: 4, text: "Most cpu quota used", value: "cpu-desc" },
+  { key: 1, text: "Latest first", value: "desc" },
+  { key: 2, text: "Oldest first", value: "asc" },
+  { key: 3, text: "Most disk used", value: "disk-desc" },
```

Comment:
Minor: we could capitalize the name and use `Disk` like we do for quota notifications, etc ([example](https://user-images.githubusercontent.com/4990025/173036195-bbd644da-5254-467a-aa7c-0b2ae4c1aaf9.png))